src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+    private static int taskNumber = 0;
+    private static Task[] t = new Task[100];
+    private static String line = "*********************************************";
+
+    private static void helloMessage() {
+        System.out.println(line);
+        System.out.println("Hello! I'm Julia");
+        System.out.println("What can I do for you?");
+        System.out.println();
+        System.out.println(line);
+    }
+    private static void byeMessage() {
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println();
+        System.out.println(line);
+        System.exit(0);
+    }
+
+    public static void printList(Task[]t){
+        int count=1;
+        System.out.println("Here are the tasks in your list: ");
+        for(int i=0; i<taskNumber;i++) {
+            System.out.println(count+"."+t[i].toString());
+            count++;
+        }
+        System.out.println(line);
+    }
+
+    public static void markAsDone(int number){
+        System.out.println("Nice! I've marked this task as done:");
+        t[number-1].markAsDone();
+        System.out.println(t[number-1].toString());
+        System.out.println(line);
+    }
+    public static void addTodo(String newTask){
+        System.out.println("Got it. I've added this task: ");
+        Todo todo = new Todo(newTask);
+        t[taskNumber] = todo;
+        taskNumber++;
+        System.out.println(todo.toString());
+        System.out.println("Now you have "+ taskNumber+ " tasks in the list.");
+        System.out.println(line);
+    }
+    public static void addDeadline(String deadline){
+        String[] split = deadline.split(" /by");
+        System.out.println("Got it. I've added this task: ");
+        Deadline day = new Deadline(split[0],split[1]);
+        t[taskNumber]=day;
+        taskNumber++;
+        System.out.println(day.toString());
+        System.out.println("Now you have "+ taskNumber+ " tasks in the list.");
+        System.out.println(line);
+
+    }
+
+    public static void addEvent(String event){
+        String[] split = event.split(" /at");
+        System.out.println("Got it. I've added this task: ");
+        Event time = new Event(split[0],split[1]);
+        t[taskNumber]=time;
+        taskNumber++;
+        System.out.println(time.toString());
+        System.out.println("Now you have "+ taskNumber+ " tasks in the list.");
+        System.out.println(line);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        helloMessage();
+        String input;
+        while(true){
+            input = in.nextLine();
+            System.out.println(line);
+            if(input.equals("bye")){
```

Comment:
This is the correct if-conditional layout. Perhaps you could consider using a "switch" statement instead?